src/Notifier.js

```diff
@@ -123,6 +123,26 @@ var Notifier = {
         return global.Notification.permission == 'granted';
     },
 
+    isPermissionDefault: function() {
+        if (!this.supportsDesktopNotifications()) return false;
+        return global.Notification.permission == 'default';
+    },
+
+    showToolbar: function() {
```

Comment:
Thanks. Sorry I missed the other PR. It's useful to say something about the other PR in the PR comments!